deps.ts

```diff
@@ -2,9 +2,9 @@ export {
   Checkbox,
   Confirm,
   Select,
-} from "https://deno.land/x/cliffy@v0.25.7/mod.ts";
-export { join } from "https://deno.land/std@0.220.1/path/posix/join.ts";
-export { dirname } from "https://deno.land/std@0.220.1/path/posix/dirname.ts";
-export { ensureDir } from "https://deno.land/std@0.220.1/fs/ensure_dir.ts";
-export * as colors from "https://deno.land/std@0.220.1/fmt/colors.ts";
-export { parseArgs } from "https://deno.land/std@0.220.1/cli/parse_args.ts";
+} from "https://deno.land/x/cliffy@v1.0.0-rc.4/mod.ts";
```

Comment:
Thanks for updating the dependencies, but I prefer to keep using `cliffy@v0.25.7` until the `v1.0.0` becomes stable.